addons/account/models/account_move.py

```diff
@@ -1864,7 +1864,9 @@ def copy(self, default=None):
             default['date'] = self.company_id._get_user_fiscal_lock_date() + timedelta(days=1)
         if self.move_type == 'entry':
             default['partner_id'] = False
-        return super(AccountMove, self).copy(default)
+        copied_am = super(AccountMove, self).copy(default)
+        copied_am.message_post(body=_('This entry has been duplicated from %s' % self.name))
```

Comment:
In theory, self.name needs html_escaping.

Reply:
a comma `,` would solve it right?

Reply:
I do not think so

Reply:
modified